per/admin.py

```diff
@@ -155,6 +155,26 @@ class FormComponentQuestionAndAnswerAdmin(admin.ModelAdmin):
     pass
 
 
+class OpsLearningAdmin(admin.ModelAdmin):
+
```

Comment:
Let's also add some other nice things for the Admin like `search_fields`, `list_filter` and `list_display` so we improve the experience around searching these in the Admin.

Reply:
Done, some example entities added – could be changed when the final structure is found out.